components/service/pocket/src/main/java/mozilla/components/service/pocket/PocketRecommendedStory.kt

```diff
@@ -7,19 +7,20 @@ package mozilla.components.service.pocket
 /**
  * A Pocket recommended story.
  *
- * @property id a unique identifier for this recommendation.
  * @property title the title of the story.
- * @property publisher the name domain of the publisher, e.g. "The New Yorker" / "nationalgeographic.co.uk"".
  * @property url a "pocket.co" shortlink for the original story's page.
  * @property imageUrl a url to a still image representing the story.
- * @property timeToRead inferred time needed to read the entire story.
+ * @property publisher optional publisher name/domain, e.g. "The New Yorker" / "nationalgeographic.co.uk"".
+ * **May be empty**.
```

Comment:
Maybe making this and `timeToRead` nullable would be a better approach on forcing users to consider the "empty"/"unknown" status.